src/routes/Missions.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import MissionsEntry from '../components/missions/missionEntry';
+import missionStyles from '../style/Missions.module.css';
+import { fetchMissions } from '../redux/missions/missionsSlice';
+
+const Missions = () => {
+  const missions = useSelector((store) => store.missions.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
```

Comment:
- [OPTIONAL] Kindly make sure to fetch the missions only once because at this moment you are fetching them every time we go to the missions page. Kindly fix that.

```suggestion
   useEffect(() => {
    if (missions.length === 0) {
      dispatch(fetchMissions());
    }
  }, [dispatch]);
```

Reply:
Okay, thank you so much @levy002